docs/usage/configuration-options.md

```diff
@@ -880,7 +880,7 @@ The above would mean Renovate would not include files matching the above glob pa
 ## extends
 
 See [shareable config presets](./config-presets.md) for details.
-Learn how to use presets by reading the [Key concepts, Presets](./key-concepts/presets.md/#how-to-use-presets) page.
+Learn how to use presets by reading the [Key concepts, Presets](./key-concepts/presets.md#how-to-use-presets) page.
```

Comment:
There's one more hit for `./md#` in the docs, but it's for the "new package manager questionnaire" that's not published on the docs site.

Reply:
can be fixed in separate PR